static/js/directives/repo-view/repo-panel-mirroring.js

```diff
@@ -39,7 +39,7 @@ angular.module('quay').directive('repoPanelMirror', function () {
       vm.robot = null;
       vm.status = null;
       vm.syncInterval = null;
-      vm.syncStartDate = null;
+      vm.syncStartDate = moment().unix();
```

Comment:
I can't remember off-hand, is there ever any case where the start date would be None/null?

Reply:
No, the start date is always set.